README.md

```diff
@@ -293,23 +349,6 @@ func ReadItemsFrom(in io.Reader) ([]Item, error) {
   }
 }
 ```
-## Notes
-
-* This package only supports text as UTF-8.  It does not support the UTF-16 or UTF-32 forms.
-* The `Float` type is limited to the minimum and maximum values that Go is able to handle with float64.  These values
-  are approximately `1.797693e+308` and `4.940656e-324`.  Values that are too large will round to infinity and
-  values that are too small will round to zero.
-
-## TODO
-
-* Symbol table construction and verification.
-* Define the external interfaces for marshalling and unmarshalling.
-* Serializing Values to the textual and binary forms.
-* Unmarshal into a struct.
-* Profiling.
-* Make the `Decimal` type keep track of precision, so trailing zeros won't be lost when translating from
-  the textual to binary form.
-* Make the `Timestamp` type handle the case of unknown local offsets.
-* Make the `Float` and `Decimal` types recognize negative zero.
-
+### License
 
+This library is licensed under the MIT License.
```

Comment:
Apache 2.0? (I'm not _necessarily_ opposed to relicensing my contributions, but if you are going to change the license I'd be interested to hear more about why.)

Reply:
Ha, good catch. There is no re-licensing. This is a copy paste error :)